docs/migrating_to_nextcord.rst

```diff
@@ -1,52 +1,52 @@
-.. currentmodule:: nextcord
-

```

Comment:
whats changed here?

Reply:
it's a dumb github thing, it's the same as the repo's current one